src/ims.js

```diff
@@ -194,11 +192,11 @@ class Ims {
    * @param {string} env The name of the environment. `prod` and `stage`
    *      are the only values supported. `prod` is default and any value
    *      other than `prod` or `stage` stage is assumed to be the default
-   *      value of `prod`.
+   *      value of `prod`. If not set, it will get the global cli env value. See https://github.com/adobe/aio-lib-env
    */
-  constructor (env) {
+  constructor (env = getCliEnv()) {
     if (!env || !IMS_ENDPOINTS[env]) {
-      env = DEFAULT_ENVIRONMENT
+      env = DEFAULT_ENV
```

Comment:
is there any reason to not set the default
```
if (!env) { env = DEFAULT_ENV }
```
directly in `getCliEnv`  in the aio-lib-env module. I fear we would need to add the extra condition everywhere we use `getCliEnv` which means an additional test for coverage.

Reply:
actually it is already DEFAULT_ENV when not set in getCliEnv(): https://github.com/adobe/aio-lib-env/blob/b506160e5a2ae9c15f313fcd5aa00c9c6c672210/src/index.js#L38

 Looks like here I kept the same if conditional statement,  when env would never be empty anyways because of the default parameter. Will fix.

Reply:
The `!IMS_ENDPOINTS[env]` still applies, since that is a guard just in case this module doesn't define the endpoint for the env...

Reply:
actually `env` can possibly be `null` so that's why we check the second expression in the if conditional.

Reply:
resolved in https://github.com/adobe/aio-lib-ims/pull/66/commits/3eac7959676aaf354b7432509cbd5edc23ce6618